src/wpt/eslint.config.mjs

```diff
@@ -0,0 +1,3 @@
+import { baseConfig } from '../../tools/base.eslint.config.mjs';
+
+export default [...baseConfig({ tsconfigRootDir: import.meta.dirname })];
```

Comment:
It seems that ts_project for WPT does not use this eslint config. We have to add eslint_bin.eslint_test just like in wd_test_bundle if we want to run eslint in this folder. Otherwise, only your IDE will complain, and it won't fail on CI.

Reply:
Thanks for the pointer! I have eslint on now and have fixed some stuff. Still have some issues to get through.